src/koboldai_client.py

```diff
@@ -43,13 +43,19 @@ def run_raw_inference_on_kai(
     repetition_penalty: float,
     **kwargs,
 ) -> str:
+    chunk_size = 32
     endpoint = f"{koboldai_url}/api/v1/generate"
     payload = {
         "prompt": prompt,
 
+        # The config.json for our models enables chat mode by default, which
+        # we don't want yet since only the dev fork can handle our model
+        # properly at the moment, and we're bundling the stable version here.
+        "chatmode": False,
```

Comment:
As it turns out this is not a param on the API, so we can't fix it like this. Closing this PR and instead reverting the KAI-related changes on the models' `config.json` files.